File: Tugas A/product.java
class product extends memory{
    private String price;
    private String id;

    // construct
    public product() {}

    public product(String price, String id){
        this.price = price;
        this.id = id;
    }
	
	//set and get
	public void setprice(String price){
		this.price = price;
	}
	public String getprice(){
		return this.price;
	}
	
	public void setid(String id){
		this.id = id;
	}
	public String getid(){
		return this.id;
	}

}

File: Tugas B/php/product.php
<?php

include "memory.php";

class product extends memory
{
	
	private $price = "";
	private $id = "";
	
	
	//Construct
	public function __construct($price = "", $id = ""){
		$this->price = $price;
		$this->id = $id;
	}
	
	//Set dan get
	public function setprice($price){
		$this->price = $price;
	}
	public function getprice(){
		return $this->price;
	}
	
	public function setid($id){
		$this->id = $id;
	}
	public function getid(){
		return $this->id;
	}
	
}
?>